src/components/ComponentCaptures/ComponentCaptures.tsx

```diff
@@ -0,0 +1,97 @@
+import { CSS_COLOR, CSS_FONT_SIZE } from '@Constants/style'
+import { graphql, useStaticQuery } from 'gatsby'
+import React, { FC, useCallback } from 'react'
+import styled from 'styled-components'
+
+const query = graphql`
+  query ComponentCaptureData {
+    allComponentCapture {
+      nodes {
+        id
+        groupName
+        storyKinds {
+          kindName
+          iframeUrl
+          thumbnailFileName
+          displayName
+          numberOfStories
+        }
+      }
+    }
+  }
+`
+
+// 一覧表示するサムネイル用画像は、`ts-node ./scripts/component-thumbnails/componentThumbnails.ts`を実行して生成します
+export const ComponentCaptures: FC = () => {
+  const { allComponentCapture } = useStaticQuery<Queries.ComponentCaptureDataQuery>(query)
+  const convertKebab = useCallback((target: string) => {
+    return target
+      .replace(/[^a-zA-Z0-9-]/g, '') // 全角文字などの半角英数字以外を除去
+      .replace(/([a-z0-9])([A-Z])/g, '$1-$2')
+      .replace(/([A-Z])([A-Z][a-z])/g, '$1-$2')
+      .toLowerCase()
+  }, [])
+
+  return (
+    <Wrapper>
+      {allComponentCapture.nodes.map((node) => (
+        <ComponentGroup key={node.groupName}>
+          <h2>{node.groupName}</h2>
+          <ComponentList>
+            {node.storyKinds.map((storyKind) => {
+              return storyKind.iframeUrl ? (
+                <li key={storyKind.iframeUrl}>
+                  <a href={convertKebab(storyKind.displayName)}>
```

Comment:
```suggestion
                  <a href={`${convertKebab(storyKind.displayName)}/`}>
```
.mdxファイル内では、サイト内リンクには末尾スラッシュをつけているので、特に事情がなければついていると良いかもしれません。
（コンポーネント内＆変数が入っているのでリンクチェッカーでは検知できないですが…）

Reply:
トップページのナビゲーションや、「前へ」「次へ」リンクのような感じで、ホバー時に色が変わるなどのインタラクションがあっても良いかも？と思いました。（すでに検討した結果でしたら無視してくださいませ）